### PR TITLE
Update Finetuning README with Alternating Flow Instructions

### PR DESCRIPTION
**Pull request by Satrat**

### CODE REVIEW

File: src/sparseml/transformers/finetune/example_alternating_recipe.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  pruning_modifiers:
+    ConstantPruningModifier:
+      targets: [
+        "re:.*self_attn.q_proj",
+        "re:.*self_attn.k_proj",
+        "re:.*self_attn.v_proj",
+        "re:.*self_attn.o_proj",
+        "re:.*mlp.gate_proj",
+        "re:.*mlp.up_proj"
+      ]
```
Comment (mgoin) [inline src/sparseml/transformers/finetune/example_alternating_recipe.yaml:53]: why can't we just use `_ALL_` here like we used previous for sparse transfer?

File: src/sparseml/transformers/finetune/example_alternating_recipe.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  run_type: oneshot
+  obcq_modifiers:
+    SparseGPTModifier:
+      sparsity: 0.7
+      block_size: 128
+      sequential_update: False
+      quantize: False
+      percdamp: 0.01
+      prunen: 0
+      prunem: 0
```
Comment (mgoin) [inline src/sparseml/transformers/finetune/example_alternating_recipe.yaml:38]: i thought this was replaced by `mask_structure: "0:0"` - does this still work? that's fine for backwards-compatibility

File: src/sparseml/transformers/finetune/example_alternating_recipe.yaml (hunk 1 above)
Comment (Satrat) [reply src/sparseml/transformers/finetune/example_alternating_recipe.yaml:53]: Yeah good point, `__ALL__` should work but let me test it quickly first

File: src/sparseml/transformers/finetune/example_alternating_recipe.yaml (hunk 2 above)
Comment (Satrat) [reply src/sparseml/transformers/finetune/example_alternating_recipe.yaml:38]: Yeah we didn't remove the args, but mask_structure is the recommended format so I'll swap it

**Review by rahul-tuli — APPROVED** [review]

**Review by bfineran — APPROVED** [review]

**Pull request merged**